KMaina/Store-Manager-API
# #161396644 Create the File Structure and Database

#### What does this PR do? 
Create the basic file structure and database tables
#### Description of Task to be completed? 
Create the file structure for the app
Create a database and populate the tables
#### How should this be manually tested?  
- Clone the repo locally 
- Ensure that `store_manager` and `test_store_manager` are created in your POSTGRESQL database
- Run the app and tables should be created
#### Any background context you want to provide? 
This is the template from which other branches will be based off 
#### What are the relevant pivotal tracker stories? 
[#161396644](https://www.pivotaltracker.com/story/show/161396644)
